src/components/login/PrivateKey.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <section id="login--saved-wallet">
-    <login-form-wrapper identifier="openPrivateKey" :on-submit="login">
+    <login-form-wrapper identifier="openPrivateKey">
```

Comment:
I think we want the on-submit rather than the on-click, so that we preserve the behavior of using the enter key.

Reply:
Very good point, sorted in [this commit](https://github.com/aphtoken/aph-desktop-web-dex/pull/9/commits/3782ab7b577f64b8c55a6dc480ae2c8f25087456). Brought up to date with master ready to merge.